watch_builds.py

```diff
@@ -0,0 +1,173 @@
+import argparse
```

Comment:
I would add a #!/usr/bin/python here and chmod +x so it can just run as a standalone script with ./watch_builds.py

Reply:
Done!